ci/appveyor-install.ps1

```diff
@@ -53,14 +53,14 @@ Exec { conda update --quiet --yes conda }
 # each about 150 MB. Enable Appveyor caching or tweak conda configuration to
 # speed up.
 Exec { conda install --channel conda-forge  --quiet --yes `
-       ixmp[tests] pytest coveralls pytest-cov }
+       ixmp[tests] "pytest>=3.9" coveralls pytest-cov }
```

Comment:
@gidden See [this log](https://ci.appveyor.com/project/gidden/ixmp/builds/26179568/job/o6hnn6rfkndwhifa). Conda was choosing to install pytest 3.2.2 on AppVeyor for Python 3 only (not Python 2.7). Shouldn't `ixmp[tests]` as a target here correspond to the 'tests' requirements in setup.py, which already specify >=3.9?